pom.xml

```diff
@@ -194,6 +194,11 @@
     <build>
         <pluginManagement>
             <plugins>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-wrapper-plugin</artifactId>
+                    <version>3.2.0</version>
+                </plugin>
```

Comment:
Why do we need this plugin now?

Reply:
It's not a plugin definition, it's a `<pluginManagement>`. We are overriding the version shipped with the maven and can use `mvn wrapper:wrapper` in the future. Otherwise, we would need always to use full plugin coordinates. 

For example in https://github.com/gradle/maven-build-scan-quickstart dependabot made the changes by specifying the full coordinates in the command line. If now one will execute `mvn wrapper:wrapper -Dmaven=3.9.0` using the latest maven 3.9.0, the wrapper version will be downgraded back to 3.1.1. See: https://scans.gradle.com/s/il4shfvrnbz7e/console-log?page=1#L8